staff/judith-gongora/cv/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <title>Cv</title>
+</head>
+<body>
+    <body>
+        <div id="header">
```

Comment:
It's better to use the header tag instead of a div. That way it's more semantic and better for SEO.